static/bidder-info/afront.yaml

```diff
@@ -0,0 +1,23 @@
+# Contact support@afront.io to connect with Afront exchange.
+# We have the following regional endpoint sub-domains:
+# US East: snt1
+# APAC: snt2
+# EU: snt3
+# Please deploy this config in each of your datacenters with the appropriate regional subdomain
+endpoint: "https://snt1.afront.io/?rsd={{.SourceId}}&sk={{.AccountID}}"
+maintainer:
+  email: "support@afront.io"
```

Comment:
Hi @Afrontio, we are ready to merge this but first we need to verify this email address is reachable. We sent an email to this address. Please respond with a message of "received".